src/python/WMCore/Storage/DeleteMgr.py

```diff
@@ -130,35 +130,36 @@ def initialiseOverride(self):
         Extract and verify that the Override parameters are all present
 
         """
-        overrideConf = self.overrideConf
-        overrideParams = {
+        
+        overrideConf = {
             "command": None,
             "option": None,
             "phedex-node": None,
             "lfn-prefix": None,
         }
 
         try:
-            overrideParams['command'] = overrideConf['command']
-            overrideParams['phedex-node'] = overrideConf['phedex-node']
-            overrideParams['lfn-prefix'] = overrideConf['lfn-prefix']
+            overrideConf['command'] = self.overrideConf['command']
+            overrideConf['phedex-node'] = self.overrideConf['phedex-node']
+            overrideConf['lfn-prefix'] = self.overrideConf['lfn-prefix']
         except Exception as ex:
-            msg = "Unable to extract Override parameters from config:\n"
-            msg += str(overrideConf)
+            msg = "Unable to extract override parameters from config:\n"
+            msg += str(self.overrideConf)
             raise StageOutInitError(msg)
-        if 'option' in overrideConf:
-            if len(overrideConf['option']) > 0:
-                overrideParams['option'] = overrideConf['option']
+        if 'option' in self.overrideConf and self.overrideConf['option'] is not None:
```

Comment:
Can we simplify this line like: `if self.overrideConf.get('option') is not None:`?

Reply:
Yes, it is much better